src-vue/src/i18n/lang/en.json

```diff
@@ -161,5 +162,12 @@
             "Northstar": "Northstar",
             "NorthstarReleaseCandidate": "Northstar release candidate"
         }
+    },
+
+    "plugins": {
+        "warning_dialog": {
+            "warning_text": "This mod contains a plugin. Plugins CAN BE REALLY DANGEROURS since they have to access to your system!",
```

Comment:
Intentional typo? :smile:

Reply:
I can't find the typo. :/

Reply:
> `DANGEROURS`

should be

> `DANGEROUS`

Reply:
fixed by [65c8997](https://github.com/R2NorthstarTools/FlightCore/pull/267/commits/65c89977cef06fff654ecb8f9b95369ee467ea95)